wowchemy/layouts/shortcodes/table.html

```diff
@@ -0,0 +1,36 @@
+{{/* Table Shortcode for Wowchemy. */}}
+{{/* Load table from page dir falling back back to remote URL */}}
+{{/* Supports delimiter separated files. The default delimiter is ',' */}}
+
+{{ $src := .Get "src" }}
+{{ $delimiter := .Get "delimiter" | default "," }}
+{{ $useHeaderRow := .Get "header" | default true }}
+{{ $caption := .Get "caption" }}
+
+{{ $is_remote := strings.HasPrefix $src "http" }}
+{{ if not $is_remote }}
+  {{ $src = path.Join  "content" $.Page.File.Dir $src }}
+{{ end }}
+{{ $rows := getCSV $delimiter $src }}
+
+<table>
+  {{ if $useHeaderRow }}
+    {{ $headerRow := index $rows 0 }}
+    {{ $rows = after 1 $rows }}
+    <tr> {{ range $headerRow }} <th>{{ . | markdownify | emojify }}</th> {{ end }} </tr>
+  {{ end }}
+  {{ range $rows }}
+    <tr>
+      {{ range . }}
+        {{ if (findRE "^\\d+$" .) }}
+          <td class="numeric">{{ . }}</td>
+        {{ else }}
+          <td>{{ . | markdownify | emojify }}</td>
```

Comment:
Please change it to `<td data-table-dtype="text">` then users can optionally use a CSS selector to format text columns.

Reply:
I've made the changes. I guess we'll leave it optional for now.